vignettes/WQPDataHarmonization.Rmd

```diff
@@ -682,7 +682,7 @@ ResultStatusIdentifier field.
 FilterFieldReview("ActivityMediaSubdivisionName", TADAProfileClean14)
 ```
 
-The ActivityMediaSubdivisionName field has two unique values, "Surface
+[Not true of current test dataset] The ActivityMediaSubdivisionName field has two unique values, "Surface
```

Comment:
Would it make sense to remove the specifics here and be more general so that it does not need updating each time we change the example data?

Reply:
That sounds great. It's hard to keep this current when we occasionally change the test dataset!

Reply:
Agreed, we can work on that in the future